### PR TITLE
tegile projects work in progress

### PR DESCRIPTION
**Pull request by theherodied**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by theherodied** [comment]
DON'T MERGE THIS YET. I'm still working on this.

File: includes/polling/storage/intelliflash2.inc.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+//$storage['size'] = ($pdst + $pfst);
+//$storage['used'] = ($pdst);
+//$storage['free'] = ($pfst);
+//
+//$storage['size'] = $pdst;
+//$storage['used'] = $pdst;
+//$storage['free'] = $pfst;
+//
+//
+$storage['size'] = (((($entry['projectDataSizeHigh'] << 32 ) + $entry['projectDataSizeLow']) * $units) + (($entry['projectFreeSizeHigh'] << 32 ) + $entry['projectFreeSizeLow']) * $units);
```
Comment (murrant) [inline includes/polling/storage/intelliflash2.inc.php]: This should be calculated after used and free: `$storage['used'] + $storage['free']`

File: includes/polling/storage/intelliflash2.inc.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+$storage['used'] = ((($entry['projectDataSizeHigh'] << 32 ) + $entry['projectDataSizeLow']) * $units);
```
Comment (murrant) [inline includes/polling/storage/intelliflash2.inc.php]: $units is null here, which is cast to an integer (0).  Also, you don't need this many parenthesis, only around the shift.
Same goes for the next line.

**Comment by murrant** [comment]
Can you update your original PR?

**Pull request closed without merging**